anime-wallpaper@flyflyan/files/anime-wallpaper@flyflyan/po/de.po

```diff
@@ -0,0 +1,106 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-12-20 15:57+0200\n"
+"PO-Revision-Date: 2024-03-06 21:52+0100\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3\n"
+"Last-Translator: Martin Posselt <nekomajin@gmx.de>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: de\n"
+
+#. settings-schema.json->saveImage->description
+#. settings-schema.json->saveImage->tooltip
+#: applet.js:89
+msgid "Automatically save background images"
+msgstr "Automatisches Speichern von Hintergrundbildern"
+
+#. settings-schema.json->slideshow->description
+#. settings-schema.json->slideshow->tooltip
+#: applet.js:93
+msgid "Play backgrounds as a slideshow"
+msgstr "Hintergrundbilder als Diaschau abspielen"
+
+#. settings-schema.json->click->description
+#. settings-schema.json->click->tooltip
+#: applet.js:97
+msgid "Enable click to switch images"
+msgstr "Mausklick zum Wechseln der Bilder aktivieren"
+
+#. metadata.json->name
+msgid "Anime Desktop Wallpaper"
+msgstr "Anime Desktop Hintergrundbilder"
+
+#. metadata.json->description
+msgid "Download and apply Anime Wallpaper to your desktop"
+msgstr "Anime Hintergrundbilder herunterladen und auf Deinem Desktop anzeigen"
```

Comment:
I think we don't use the informal "Du" form anywhere. So I'd avoid it with a construct like:

```suggestion
msgstr "Anime Hintergrundbilder herunterladen und auf dem Desktop anzeigen"
```